src/content_script/mastodonInject.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+/**
+ * Replacement onClick handler for Follow button.
+ * @param {Event} event 
```

Comment:
```suggestion
 *
 * @param {Event} event 
```

As said everywhere please